content/en/spec/activitypub.md

```diff
@@ -41,10 +41,10 @@ QuoteRequest
 
 The first-class Object types supported by Mastodon are `Note` and `Question`.
 
-- Notes are transformed into regular statuses.
-- Questions are transformed into a poll status. See the [Polls](#Question) extension for more information.
+- Mastodon transforms Notes into regular statuses.
```

Comment:
this is less readable; the bullet points should start with the type. consider changing the text above the list instead, although it already clarifies the context of Mastodon